### DISCUSSION TITLE
Is there any way to calculate the proportion of my asset structure?

### DISCUSSION THREAD

**Discussion by hiikii** [firefly-iii/firefly-iii#8796] [Firefly III] [answered]
Hello everyone,

I am a new user of firefly and I am still trying to get familiar with it. Now I have a problem and don't know how to solve it.

When I try to calculate the statistics of my account, I seem to be able to only see intuitive numbers but no way to know their proportion in my overall assets. I want to know if I have missed something?

To be honest, I am trying to add my time deposits and funds to my account, so I will be more concerned about their proportion of my assets.

If anyone knows how to do this please let me know, I'd be very grateful.

**Comment by JC5** [comment, marked as answer]
Sorry, I think I know what you mean but I don't have that feature in Firefly III.

**Reply by hiikii** [reply]
Hello,

Thanks for your answer, I thought I did something wrong.

I don't know if this is a necessary feature, but if possible, I'd love it to be implemented, just like seeing the percentage of income and expenses in the report.

Anyway, thank you for developing firefly III, which gave me the courage to ditch proprietary software and travel with my data.
